src/modules/Builder/features/ActivitySettings/SubscalesConfiguration/SubscaleLookupTable/SubscaleLookupTable.const.tsx

```diff
@@ -0,0 +1,26 @@
+import i18n from 'i18n';
+
+const { t } = i18n;
+
+export const columnData = [
+  {
+    key: 'Score',
+    label: t('Score'),
+  },
+  {
+    key: 'Raw Score',
+    label: t('Raw Score'),
```

Comment:
It seems here has to be t('rawScore') and in other labels key need to be with small first letter, for example t('score')